bin/ExtractGName.R

```diff
@@ -7,7 +10,7 @@ zip.fname <- paste(fname, 'zip', sep='.')
 heatmap.dat <- file.path(fname, 'heatmap.RData')
 load(unz(zip.fname, heatmap.dat))
 
-gene.list <- strsplit(go.list[[1]], ':')
-gname.list <- sapply(gene.list, function(x) x[1])
+gene.list <- as.data.frame(go.list, stringsAsFactors = FALSE)
```

Comment:
I will try to rethink how to do this in the simplest means possible.

It would also be possible to have the other algorithms return  go.list[[2]] with the same size as  go.list[[1]]. i.e. simplest way would be to return the gene order twice.